M3-JavaScriptBasics/assets/js-course-m3-TarasYarovyi-home-task.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+
+        <h2>JavaScript Mentoring Program - Java Script Basics</h2>
+        <p id="home-task">Here we have some text!</p>
+        <button type="button" onclick="MyFunction()">Try it</button>
+    </body>
+</html>
+<script type="text/javascript"> 
+    alert("This alert box was called with the onload event");
+    function MyFunction() {
+        document.getElementById("home-task").innerHTML = Date();
```

Comment:
The most valuable ability of any programmer is the ability to search on the internet, 
By writing appropriate requests in Google you will be able to find 50% of all solutions you are interested in. 
For instance, your task is: `current date and time in format DD-MM-YYYY HH:MM:SS` so just by searching it
you will be able to find this: https://stackoverflow.com/questions/10211145/getting-current-date-and-time-in-javascript

Reply:
Fixed